static/app/components/repositoryProjectPathConfigForm.tsx

```diff
@@ -26,6 +29,15 @@ type Props = {
   existingConfig?: RepositoryProjectPathConfig;
 };
 
+interface IntegrationRepo {
+  defaultBranch: string;
+  /**
+   * ex - getsentry/sentry
+   */
+  identifier: string;
+  name: string;
```

Comment:
What's the difference?

Reply:
moved it and added a comment, we have a `org/repos/ `endpoint and we have `org/integration/:integrationId/repos/`